tests/application/cms/test_utils.py

```diff
@@ -57,38 +58,42 @@ def setup(self):
     def teardown(self):
         current_app.config = {**self.saved_config}
 
-    def test_returns_two_data_source_forms(self, stub_measure_page):
-        form1, form2 = get_data_source_forms(request, stub_measure_page)
+    def test_returns_two_data_source_forms(self):
+        measure_version = MeasureVersionFactory()
+        form1, form2 = get_data_source_forms(request, measure_version)
 
         assert isinstance(form1, DataSourceForm)
         assert isinstance(form2, DataSourceForm)
 
-    def test_returned_forms_have_distinct_prefixes(self, stub_measure_page):
-        form1, form2 = get_data_source_forms(request, stub_measure_page)
+    def test_returned_forms_have_distinct_prefixes(self):
+        measure_version = MeasureVersionFactory()
+        form1, form2 = get_data_source_forms(request, measure_version)
 
         assert form1._prefix == "data-source-1-"
         assert form2._prefix == "data-source-2-"
 
     @pytest.mark.parametrize("csrf_enabled", [True, False])
-    def test_csrf_enabled_depending_on_app_config(self, stub_measure_page, csrf_enabled):
+    def test_csrf_enabled_depending_on_app_config(self, csrf_enabled):
+        measure_version = MeasureVersionFactory()
         current_app.config["WTF_CSRF_ENABLED"] = csrf_enabled
 
-        form1, form2 = get_data_source_forms(request, stub_measure_page)
+        form1, form2 = get_data_source_forms(request, measure_version)
 
         assert form1.meta.csrf is csrf_enabled
         assert form2.meta.csrf is csrf_enabled
 
-    def test_csrf_disabled_if_sending_to_review(self, stub_measure_page):
+    def test_csrf_disabled_if_sending_to_review(self):
+        measure_version = MeasureVersionFactory()
         assert current_app.config["WTF_CSRF_ENABLED"] is False
```

Comment:
This test might be broken - this should probably set it to `True` on the app and then check it's still false when sending to review?

Reply:
Good spot.